test/versioned/node-rdkafka/kafka.tap.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2024 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const tap = require('tap')
+const helper = require('../../lib/agent_helper')
+const { removeModules } = require('../../lib/cache-buster')
+
+tap.beforeEach(async (t) => {
+  t.context.agent = helper.instrumentMockedAgent()
+
+  const Kafka = require('node-rdkafka')
+  t.context.Kafka = Kafka
+
+  await new Promise((resolve) => {
+    const producer = new Kafka.Producer({
+      'metadata.broker.list': '127.0.0.1:9092'
```

Comment:
we put the host/port info in `test/lib/params.js`

Reply:
Why? I see that the file introduces a bunch of environment variables for overriding things, but what utility are they providing?